cmd/ignited/cmd/gitops.go

```diff
@@ -27,7 +27,7 @@ type gitOpsFlags struct {
 	username     string
 	password     string
 
-	paths []string
+	// paths []string
```

Comment:
This is not used anywhere. Remove it? or leave it commented?

Reply:
I'd say remove it if it's indeed completely unused.